### DISCUSSION TITLE
Version update problem

### DISCUSSION THREAD

**Discussion by wuhuwdnmd21** [MartinTheDragon/Nuclear-Tech-Mod-Remake#23] [General]
Author, I like this mod very much, but I always play with version 1.16.5, so can you update version 1.16.5

**Comment by MartinTheDragon** [comment]
I discontinued the 1.16.5 port in favor of the increased world height and cave generation 1.18 has to offer. Unless someone else decides to work on a 1.16 port, it's basically dead. The difference in modding between these two versions is rather big, so in the end it'd be pretty much twice the work. No can do, sorry.

**Reply by wuhuwdnmd21** [reply]
okay, thank you

**Reply by wuhuwdnmd21** [reply]
Author, I have another question. Why can't I see the mushroom cloud in the 1.16.5 fat man? Is there any solution

**Reply by wuhuwdnmd21** [reply]
I will not add rendering for nuclear explosions

**Reply by MartinTheDragon** [reply]
The mushroom cloud was added right before the switch to 1.18, but never released.
If you want to see it in 1.16.5, you'll have to compile a version yourself.
It's pretty easy:
- First, switch to the `1.16` branch on GitHub. The dropdown menu for it should be somewhere on the left side if you view the main page
- Then click on the green button that says `Code` and select `Download ZIP`
- When you're done downloading the archive, extract the ZIP and open a command prompt or PowerShell inside the new folder and type `.\gradlew build`
- Gradle will now compile a version for you, and how long it'll take depends on your internet connection
- When Gradle outputs `BUILD SUCCESSFUL`, navigate to the folder `build/libs` and inside should be a mod jar.
- Enjoy

**Reply by wuhuwdnmd21** [reply]
Okay, I see
